package.json

```diff
@@ -120,7 +120,7 @@
     "yargs": "~13.3.2"
   },
   "engines": {
-    "node": ">=16"
+    "node": ">=20.6.1"
```

Comment:
I think we should use node 18 and above as the engine cuz 18 will still be supported in maintenance LTS for another year or so, and customers may not be on 20 yet. I realized that `yarn` will actually error on install if your version is out of date with a package's engine, but `npm` will just give you a warning

I think we should keep using 20 in the nvmrc and CI tho